airbyte-ci/connectors/pipelines/pipelines/helpers/git.py

```diff
@@ -74,9 +72,30 @@ async def get_modified_files_in_commit(current_git_branch: str, current_git_revi
         return await get_modified_files_in_commit_remote(current_git_branch, current_git_revision)
 
 
+def get_modified_files_since_local(current_git_revision: str, since: str) -> Set[str]:
```

Comment:
I'd appreciate `since` to be a `datetime` / `date` object and make the function perform a `strftime` to cast it to string.